pkg/terminal/command.go

```diff
@@ -1063,10 +1068,49 @@ func setBreakpoint(t *Term, ctx callContext, tracepoint bool, argstr string) err
 	return nil
 }
 
+func setExecutionPoint(t *Term, ctx callContext, argstr string) error {
+	args := strings.SplitN(argstr, " ", 2)
```

Comment:
locspecs can contain spaces the better thing to do is to use all the argstr as locspec and check that it's `!= ""`